app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -112,12 +109,12 @@ def request_enrollment_code(unique_id)
     # @return [String] the token
     def retrieve_token!
       body = request_token
-      @token_expires_at = Time.zone.now + body['expires_in']
-      @token = "#{body['token_type']} #{body['access_token']}"
+      @@token_expires_at = Time.zone.now + body['expires_in']
+      @@token = "#{body['token_type']} #{body['access_token']}"
     end
 
     def token_valid?
-      @token.present? && @token_expires_at.present? && @token_expires_at.future?
+      @@token.present? && @@token_expires_at.present? && @@token_expires_at.future?
```

Comment:
Since we're defining a reader, would it make sense to drop the `@@` ?

```suggestion
      token.present? && token_expires_at.present? && token_expires_at.future?
```